src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private final Random random = new Random();
+
+    public enum Color {
+        RED,
+        WHITE,
+        BLACK,
+        BLUE,
+        YELLOW,
+        GREEN
+    }

```

Comment:
It's NOT a mistake but I should aware you that using enums as inner classes may lead to additional code rework in case parent class is no longer required and should be deleted but enum is still used somewhere else in the code